js/todo.js

```diff
@@ -0,0 +1,215 @@
+'ues strict';
+const $todoInput = document.querySelector('.new-todo');
+const $todoList = document.querySelector('.todo-list');
+const $filterList = document.querySelector('.filters');
+
+const filterNames = ['all', 'active', 'completed'];
+
+let todos = [];
+
+function init() {
+  loadTodoList();
+  showCount($filterList.querySelector('.all'));
+
+  $todoInput.addEventListener('keyup', event => {
+    addTodoItem(event);
+  });
```

Comment:
이벤트에 등록된 ```addTodoItem```함수에 전달되는 인자가 ```event``` 밖에 없기 때문에 이 부분은 인라인 함수로 등록할 필요가 없어보입니다!

```suggestion
  $todoInput.addEventListener('keyup', addTodoItem);
```

Reply:
이 부분도 바로 수정해보겠습니다!!👍